_posts/2021-10-14-Subnero-at-Ocean-Business.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Subnero attended Ocean Business 2021 in Southampton, UK
+banner : images/banner-pulse-standard.jpg
+date:  2021-10-14
+categories: wnc news
+thumbnail: images/pulse-thumbnail-ocean-business-2021.jpg
+---
+
+Subnero attended its first in-person exhibition during [Ocean Business](https://noc-events.co.uk/ocean-business-2021){:target="_blank"}, in Southampton, UK.
+
+The event took place at the National Oceanography Centre from the 12th to the 14th of October 2021, after being postponed because of the pandemic. 
+
+It was so nice to meet for the team to talk about our technology in person.
```

Comment:
```suggestion
It was so nice to meet the team to talk about our technology in person.
```

Reply:
There is a typo here "meet" should be there. I'll correct the entire sentence.